dags/openshift_nightlies/tasks/index/status.py

```diff
@@ -51,10 +31,14 @@ def __init__(self, dag, version, release_stream, latest_release, platform, profi
         # Upstream task this is to index
         self.task = task 
         self.env = {
-            "OPENSHIFT_CLIENT_LOCATION": self.latest_release["openshift_client_location"],
             "RELEASE_STREAM": self.release_stream,
             "TASK": self.task
         }
+        self.git_user = var_loader.get_git_user()
+        if self.git_user == 'cloud-bulldozer'
+            self.env["ES_INDEX"] = "perf_scale_ci"
+        else:
+            self.env["ES_INDEX"] = f"{self.git_user}_playground"
```

Comment:
when we spin up a playground does it create this index?

Reply:
it will the first time a task is indexed